### ISSUE TITLE
Broken Planetary view mode

### ISSUE THREAD

**Issue by MMikado1** [Touhma/DSP_Plugins#189]
Since the patch version 0.9.27.15033 (yesterday when i write this), the ressource information panel in the planetary view mode and starmap view mode doesn't appear anymore. An error message appear and we can't see how many ressources are available on each planet, which make the game  a little bit more difficult when we are trying to find what we want.
I didn't find any other problem within this patch after playing a couple of hours.

**Comment by adaliszk** [comment]
The exception thrown:
```ini
System.MissingMethodException: bool PlanetData.CalcVeinAmounts(long[]&)
  at UIPlanetDetail.OnPlanetDataSet () [0x0001c] ;IL_001C 
  at UIPlanetDetail.set_planet (PlanetData value) [0x00010] ;IL_0010 
  at UIGame.SetPlanetDetail (PlanetData planet) [0x0001c] ;IL_001C 
  at UIGame._OnUpdate () [0x00e96] ;IL_0E96 
  at ManualBehaviour._Update () [0x0001f] ;IL_001F 
```

**Comment by vanth0me** [comment]
Can confirm. Disabled all other mods and it is indeed Galactic Scale.

**Comment by innominata** [comment]
Thanks for the report, should be fixed in 2.8.6
